ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -221,6 +223,7 @@ public void testConcurrentPutsSuccess()
     for (int i = 0; i < 5; i++) {
       requestAndResultsList.add(new RequestAndResult(chunkSize + random.nextInt(5) * random.nextInt(chunkSize)));
     }
+    instantiateNewRouterForPuts = false;
```

Comment:
I don't see how this statement contravenes that comment, it enables it on the contrary. 

The test is only trying to make more puts on the same router. It is not trying to ensure that the router does not get closed after the first set of puts - which would be an arbitrary check as far as this test is concerned.